src/services/requestForm.service.js

```diff
@@ -1,16 +1,46 @@
-const URL = "https://haifa-dev.herokuapp.com";
+const URL = "https://haifa-dev.herokuapp.com/api/v1";
 
-export async function submitForm(isForProfit) {
+const errorMessages= {
+    name: "You forgot to fill your name!",
+    email: "How can we contact you without an email?",
+    phone: "How can we contact you without a phone number?",
+    about: "You forgot to provide details about your project!",
+    description: "You forgot to provide details about your organization!",
+    webAddress: "You should never see this error message. Like, never.",
+    tasks: "You haven't told us what needs to be done!"
+};
+
+function mapErrorMessage(errorMsg) {
+    if (errorMsg.startsWith('"')) {
```

Comment:
Not very robust. Would be better to match expected response from server.
Also, you forgot to handle 'internal server error' message that the server can respond with.